charts/aws-efs-csi-driver/templates/node-daemonset.yaml

```diff
@@ -11,6 +11,10 @@ spec:
       app: efs-csi-node
       app.kubernetes.io/name: {{ include "aws-efs-csi-driver.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
+  {{- with .Values.node.updateStrategy }}
+  strategy:
```

Comment:
This is incorrect. The field name for DaemonSets is `updateStrategy`, not `strategy`

Reply:
good catch, indeed `strategy` is for Deployments and for daemonset should be `updateStrategy`. will fix it soon

Reply:
@617m4rc  fixed, can you please review and resolve if you agree?